src/pyclts/transcriptiondata/creanza.tsv

```diff
@@ -0,0 +1,335 @@
+glottalized aspirated voiceless lateral click consonant	ǁʰˀ	281	ǁˀʰ
+voiceless palatal click consonant	ǂ	186	ǂ
+aspirated voiceless palatal click consonant	ǂʰ	177	ǂʰ
+breathy voiceless palatal click consonant	ǂʱ	210	ǂʱ
+glottalized voiceless palatal click consonant	ǂˀ	284	ǂˀ
+glottalized aspirated voiceless palatal click consonant	ǂʰˀ	234	ǂˀʰ
+central unrounded nearly-open central vowel	ɐ	145	ɐ
+back unrounded open back vowel	ɑ	139	ɑ
+non-syllabic back unrounded open back vowel	ɑ̯	427	ɑ̯
+back rounded open back vowel	ɒ	134	ɒ
+voiced bilabial implosive consonant	ɓ	602	ɓ
+labialized voiced bilabial implosive consonant	ɓʷ	671	ɓʷ
+back rounded open-mid back vowel	ɔ	137	ɔ
+back breathy rounded open-mid back vowel	ɔ̤	127	ɔ̤
+voiced dental implosive consonant	ɗ	697	ɗ
+central unrounded mid central vowel	ə	148	ə
+central breathy unrounded mid central vowel	ə̤	153	ə̤
+non-syllabic central unrounded mid central vowel	ə̯	417	ə̯
+front unrounded open-mid front vowel	ɛ	410	ɛ
+front breathy unrounded open-mid front vowel	ɛ̤	409	ɛ̤
+voiced palatal stop consonant	ɟ	606	ɟ
+voiced velar implosive consonant	ɠ	639	ɠ
+voiced velar stop consonant	g	662	ɡ
+voiced alveolar click consonant	gǃ	263	ɡ!
+glottalized aspirated voiced alveolar click consonant	gǃʰˀ	250	ɡ!ˀʰ
+voiced alveolar nasal consonant	n	156	ɡ/n
+from voiced velar stop to voiced bilabial stop cluster	gb	576	ɡb
+from voiced velar stop to labialized voiced bilabial stop cluster	gbʷ	663	ɡbʷ
+from voiced velar stop to voiced alveolar stop cluster	gd	578	ɡd
+voiced lateral click consonant	gǁ	249	ɡǁ
+glottalized aspirated voiced lateral click consonant	gǁʰˀ	286	ɡǁˀʰ
+voiced palatal click consonant	gǂ	235	ɡǂ
+glottalized aspirated voiced palatal click consonant	gǂʰˀ	224	ɡǂˀʰ
+voiced labial click consonant	gʘ	258	ɡʘ
+palatalized voiced velar stop consonant	gʲ	677	ɡʲ
+labialized voiced velar stop consonant	gʷ	610	ɡʷ
+glottalized voiced velar stop consonant	gˀ	670	ɡˀ
+long voiced velar stop consonant	gː	691	ɡː
+labialized long voiced velar stop consonant	gʷː	590	ɡːʷ
+creaky voiced velar stop consonant	g̰	699	ɡ̰
+voiced velar fricative consonant	ɣ	303	ɣ
+labialized voiced velar fricative consonant	ɣʷ	295	ɣʷ
+back unrounded close-mid back vowel	ɤ	131	ɤ
+voiced labialized-palatal approximant consonant	ɥ	432	ɥ
+voiced glottal fricative consonant	ɦ	354	ɦ
+central unrounded close central vowel	ɨ	150	ɨ
+central breathy unrounded close central vowel	ɨ̤	144	ɨ̤
+non-syllabic central unrounded close central vowel	ɨ̯	422	ɨ̯
+voiceless alveolar lateral fricative consonant	ɬ	444	ɬ
+aspirated voiceless alveolar lateral fricative consonant	ɬʰ	434	ɬʰ
+glottalized voiceless alveolar lateral fricative consonant	ɬˀ	440	ɬˀ
+long voiceless alveolar lateral fricative consonant	ɬː	445	ɬː
+voiceless dental lateral fricative consonant	ɬ̪	457	ɬ̪
+voiced alveolar lateral fricative consonant	ɮ	454	ɮ
+back unrounded close back vowel	ɯ	128	ɯ
+back unrounded close back with_rising_tone vowel	ɯ̌	138	ɯ̌
```

Comment:
I could bet that Ruhlen meant this vowel to be "ultra-short", but the symbol is not the right one, it is the official symbol for a tone ;)